tModCodeAssist/tModCodeAssist.sln

```diff
@@ -0,0 +1,37 @@
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
+# Visual Studio Version 17
+VisualStudioVersion = 17.10.34607.79
+MinimumVisualStudioVersion = 10.0.40219.1
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "tModCodeAssist", "tModCodeAssist\tModCodeAssist.csproj", "{DFEF10D8-C458-44D4-8BBB-623A516EE362}"
+EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "tModCodeAssist.CodeFixes", "tModCodeAssist.CodeFixes\tModCodeAssist.CodeFixes.csproj", "{894B3F00-E69A-4759-8B52-8F6D1B029280}"
+EndProject
```

Comment:
There's actually a good reason for that. Code Fixers use API that analyzers don't have, so if analyzers are ran then it will error and everything will implode